keras/models.py

```diff
@@ -5,7 +5,6 @@
 from . import backend as K
 from .engine.training import Model
 from .engine.topology import get_source_inputs, Node
-from .legacy.models import Graph
```

Comment:
That's not an unused statement. Without it you can't import Graph from keras.models.